go.mod

```diff
@@ -16,3 +17,5 @@ require (
 	github.com/tetratelabs/getenvoy-package v0.0.0-20190718134531-9487f25b3273
 	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8 // indirect
 )
+
+replace github.com/tetratelabs/getenvoy-package => ../getenvoy-package
```

Comment:
Q: is this on purpose?

Reply:
Ah, nope!

This was so I could use my updates from the local file directory. It won't pass lint until I remove it so when the other PR is merged I'll fix.